packages/go-gen/Cargo.toml

```diff
@@ -8,8 +8,8 @@ publish = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cosmwasm-std = { path = "../std", version = "2.0.1", features = ["cosmwasm_2_0", "staking", "stargate"] }
-cosmwasm-schema = { path = "../schema", version = "2.0.1" }
+cosmwasm-std = { path = "../std", features = ["cosmwasm_2_0", "staking", "stargate"] }
+cosmwasm-schema = { path = "../schema" }
```

Comment:
I'm not 100% sure how cargo handles this. Don't we need the version key for publishing?

Reply:
Just checked again, it fails.. I could have sworn it worked at some point that it would just source the version from that path..

Eh, whatever, gonna readd them